zipline/finance/performance/tracker.py

```diff
@@ -281,62 +281,56 @@ def to_dict(self, emission_type=None):
 
         return _dict
 
-    def process_event(self, event):
-
-        if event.type == zp.DATASOURCE_TYPE.TRADE:
-            # update last sale
-            self.position_tracker.update_last_sale(event)
-
-        elif event.type == zp.DATASOURCE_TYPE.TRANSACTION:
-            # Trade simulation always follows a transaction with the
-            # TRADE event that was used to simulate it, so we don't
-            # check for end of day rollover messages here.
-            self.txn_count += 1
-            self.position_tracker.execute_transaction(event)
-            for perf_period in self.perf_periods:
-                perf_period.handle_execution(event)
+    def process_trade(self, event):
+        self.position_tracker.update_last_sale(event)
 
-        elif event.type == zp.DATASOURCE_TYPE.DIVIDEND:
-            log.info("Ignoring DIVIDEND event.")
+    def process_transaction(self, event):
 
-        elif event.type == zp.DATASOURCE_TYPE.SPLIT:
-            leftover_cash = self.position_tracker.handle_split(event)
-            if leftover_cash > 0:
-                for perf_period in self.perf_periods:
-                    perf_period.handle_cash_payment(leftover_cash)
+        self.txn_count += 1
+        self.position_tracker.execute_transaction(event)
+        for perf_period in self.perf_periods:
+            perf_period.handle_execution(event)
 
-        elif event.type == zp.DATASOURCE_TYPE.ORDER:
-            for perf_period in self.perf_periods:
-                perf_period.record_order(event)
+    def process_dividend(self, dividend):
```

Comment:
Looks like it I omitted it without realizing. In part, because we only do `log.info("Ignoring DIVIDEND event.")`, but I'll add it to the tradesimulation loop for completion sake.